src/main.js

```diff
@@ -93,7 +94,16 @@ module.exports = {
       }
     }))
 
-    this.subscriptions.add(atom.commands.add('atom-text-editor', {
+    // Marks each JavaScript editor with a CSS class so that
+    // we can enable commands only for JavaScript editors.
```

Comment:
This linter supports any file type the user desires since there are ESLint processors for some pretty strange templating languages out there. `JavaScript` should be changed to `supported`.